README.md

```diff
@@ -200,6 +201,12 @@ Optionally, you may write the SVID and key to disk with `-write` in order to exa
     #          16:87:b2:97:2f:98:ed:80:2a:5e:62:f4:7f:87:82:ff:67:96:
     #          e6:2e:fa:a1
 
+## Using SPIRE with Envoy
```

Comment:
Nit: I feel like this should be one level higher, since it kind of assumes that you have already gotten started